sdks/python/setup.py

```diff
@@ -354,7 +354,8 @@ def get_portability_package_data():
           'fastavro>=0.23.6,<2',
           'fasteners>=0.3,<1.0',
           # TODO(https://github.com/grpc/grpc/issues/37710): Unpin grpc
-          'grpcio>=1.33.1,<2,!=1.48.0,!=1.59.*,!=1.60.*,!=1.61.*,!=1.62.0,!=1.62.1,<1.66.0',  # pylint: disable=line-too-long
+          'grpcio>=1.33.1,<2,!=1.48.0,!=1.59.*,!=1.60.*,!=1.61.*,!=1.62.0,!=1.62.1,<1.66.0; python_version <= "3.12"',  # pylint: disable=line-too-long
+          'grpcio>=1.67.0; python_version >= "3.13"', 
```

Comment:
To work around https://github.com/grpc/grpc/issues/39113 , we need to set a variable GRPC_EXPERIMENTS="-event_engine_client"  in the container image. However custom container users might not set it and will be affected by the issue. I asked GRPC team on fix ETA on the internal bug. To reduce customer friction,  we can add a check to see whether GRPC_EXPERIMENTS has been set and print a warning if it hasn't  been set and GRPC  version is affected by the regression. Hopefully there will be a fix soon though.

Reply:
Okay cool, I'll include that in the Dockerfile when I add all of the support changes. Do we know if that environment variable has any impact for GRPC versions < 1.66.0? Conditional environmental variables are not a thing docker likes to do.

Reply:
Having followed up in the grpc code base it looks like this experiment has been around a while and may have some sort of impact (specifically on macos environments as stated in grpc/grpc#33279) so I'll have to do some testing around what what does/does not work.